protocol/signalfx/spanobfuscation/tagmatchrule.go

```diff
@@ -24,7 +24,7 @@ type TagMatchRuleConfig struct {
 }
 
 func getRules(ruleConfigs []*TagMatchRuleConfig) ([]*rule, error) {
-	var rules []*rule
+	var rules = make([]*rule, 0)
```

Comment:
Can we allocate cap = len(ruleConfigs) ?

Reply:
will do